conctr.agent.class.nut

```diff
@@ -117,6 +118,9 @@ class Conctr {
                 // Set the time stamp if not set already
                 if (!("_ts" in v) || (v._ts == null)) {
                     v._ts <- time();
+                }else if(("_ts" in v) && typeof v._ts == "number" && ((v._ts.tostring().len() <= 10) ? v._ts < MIN_TIME : v._ts < MIN_TIME*1000)){
```

Comment:
Get the formatting/spacing right - space before `else` and after `if`.
Put brackets around the middle `==` statement.
The third statement there needs to be simplified or moved out into a variable. It's way too complex.
The log should say "Setting to time()".